hydromt_wflow/wflow.py

```diff
@@ -4425,13 +4437,18 @@ def setup_domestic_demand_from_population(
     def setup_other_demand(
         self,
         demand_fn: str | Dict[str, Dict[str, Any]] | xr.Dataset,
-        variables: list = ["ind_gross", "ind_net", "lsk_gross", "lsk_net"],
+        variables: list = [
+            "industry_gross",
```

Comment:
Nice! You still need to change the docstrings above and below though (required variables list)